### PR TITLE
Feature 3.10/computed values add test

### PR DESCRIPTION
**Pull request by cpjulia**
### Scope & Purpose

3.10 backport of https://github.com/arangodb/arangodb/pull/16619.

- [ ] :hankey: Bugfix
- [x] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [x] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [x] **integration tests**
  - [ ] **resilience tests**
- [x] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)

#### Related Information

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket:
- [ ] Design document:

### CODE REVIEW

File: tests/js/common/shell/shell-collection-computed-values.js [hunk 1]
```diff
@@ -72,6 +72,50 @@ function ComputedValuesAfterCreateCollectionTestSuite() {
...
+      for (let i = 0; i < 50; ++i) {
+        collection.insert({name: {first: `a${i}`, last: `b${i}`}});
+      }
+      for (let i = 0; i < 50; ++i) {
+        collection.insert({name: {first: i, last: i}});
+      }
+      let amountNullValues = 0;
+      const docs = collection.toArray();
+      docs.forEach((doc, index) => {
+        assertTrue(doc.hasOwnProperty("newValue"));
```
Comment (jsteemann) [inline tests/js/common/shell/shell-collection-computed-values.js:110]: This is not deterministic, as `toArray()` does not guarantee any order for the returned results.

**Review by jsteemann — APPROVED** [review]
LGTM

**Review by KVS85 — APPROVED** [review]
LGTM.

**Comment by KVS85** [comment]
- [x] https://jenkins01.arangodb.biz/job/arangodb-matrix-pr/22796/

**Pull request merged**